spec/mailers/user_mailer_spec.rb

```diff
@@ -18,7 +18,25 @@
     end
 
     it 'assigns @code' do
-      expect(mail.body.encoded).to match('123123')
```

Comment:
new email template insert  `space` for display code clearly, so change the spec.
 ```
old: 123123
new 123 456
 ```